public/manifest.json

```diff
@@ -1,6 +1,10 @@
 {
   "short_name": "Zeeguu (1.0)",
```

Comment:
In my phone, this is used for the icon name, wouldn't it be better to swap these around, name: Zeeguu (1.0) and short_name just Zeeguu? or just both of them being Zeeguu?